common/changes/office-ui-fabric-react/evsung-textfieldreadonly_2018-06-14-21-24.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Adding readOnly prop to textField",
+      "type": "patch"
```

Comment:
New props are backwards compatible but not forward compatible, so this should be a minor change